docs/source/platform-features/message-bus/vip/vip-known-identities.rst

```diff
@@ -7,25 +7,46 @@ VIP Known Identities
-+----------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| platform.driver      | devices               | The default identity for the Master Driver Agent.                                                                                                                                                                                                                  |
-+----------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| VIP Identity           | Sphere of Influence   | Notes                                                                                                                                                                                                                                                              |
++========================+=======================+====================================================================================================================================================================================================================================================================+
+| platform               | Platform              |                                                                                                                                                                                                                                                                    |
++------------------------+-----------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| platform.agent         | Platform              | The PlatformAgent is responsible for this identity.  It is used to allow the VolttronCentralAgent to control and individual platform                                                                                                                               |
```

Comment:
I think you should add a column that specifies the agent rather than having "The x agent is responsible for this identity"